airflow/provider.yaml.schema.json

```diff
@@ -210,7 +210,20 @@
     "additional-extras": {
       "type": "object",
       "description": "Additional extras that the provider should have"
+    },
+    "task-decorators": {
+        "type": "array",
+        "description": "Apply custom decorators to the TaskFlow API. Can be accessed by users via '@task.<name>'",
```

Comment:
Reword to say something along the lines of "Decorators to use with TaskFlow API"

Reply:
```suggestion
        "description": "Decorators to use with the TaskFlow API. Can be accessed by users via '@task.<name>'",
```